person.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'nameable'
+
+# Class representing an individual with attributes such as name, age, and parent_permission.
+class Person < Nameable
+  attr_accessor :age
```

Comment:
- [ ] The `name` and `age` should have setters and getters, kindly make sure to add them to the attr_accessor